portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -89,7 +87,7 @@ exports.create = async (req, res) => {
     emailVariables,
   );
 
-  return res.status(200).send(createdFeedback);
+  return res.status(200).send({ _id: createdFeedback.insertedId });
```

Comment:
`200` feels wrong here `201` more suitable since were changing the response data IMO